components/badge/bagde.css

```diff
@@ -0,0 +1,55 @@
+.badge-container {
+  position: relative;
+}
+
+.badge {
+  position: absolute;
+  top: -5px;
+  right: -5px;
+  color: #fff;
```

Comment:
you could use your color variable here instead.

Reply:
Thanks for pointing that out Yash, I missed that.